### ISSUE TITLE
Cannot mount new volume after another volume plugin is deleted

### ISSUE THREAD

**Issue by luomiao** [vmware/docker-volume-vsphere#1726]
How to reproduce:

1. Install vsphere volume plugin: 
`docker plugin install --grant-all-permissions --alias vsphere vmware/docker-volume-vsphere:latest`

2. Install another volume plugin:
`docker plugin install --grant-all-permissions --alias netapp netapp/ndvp-plugin:17.07`

3. Disable and rm the second volume plugin:
```
docker plugin disable netapp
docker plugin rm netapp
```

4. Create and use a volume:
```
docker volume create --driver=vsphere testVol
docker run --rm -it -v testVol:/mnt/myvol busybox
```

will give error:
```
docker: Error response from daemon: oci runtime error: container_linux.go:247: starting container process caused "process_linux.go:359: container init caused \"rootfs_linux.go:54: mounting \\\"/var/lib/docker/plugins/9e81f4468bc4c45e080e59f4fa1d3e6c0d9bd530e100b033d06b56f638ab90c9/rootfs/mnt/vmdk/testVol@vsanDatastore\\\" to rootfs \\\"/var/lib/docker/aufs/mnt/3111832daba9576dd7c81cfc6e0dcca0b162488d6a7567dadba36ce5224a3fa2\\\" at \\\"/mnt/myvol\\\" caused \\\"stat /var/lib/docker/plugins/9e81f4468bc4c45e080e59f4fa1d3e6c0d9bd530e100b033d06b56f638ab90c9/rootfs/mnt/vmdk/testVol@vsanDatastore: no such file or directory\\\"\"".
ERRO[0002] error getting events from daemon: context canceled
```

Error is gone when vsphere plugin is re-installed.

**Comment by govint** [comment]
@luomiao can you also upload plugin logs. The error, seems to say that the source volume location doesn't exist. Plugin should have mounted on that path. 

Docker issue 33378 may be related - plugin looses propagated mount and hence docker can't see the plugin mounted volume.

**Comment by govint** [comment]
Related issue is fixed in docker in 17.06.0 RC2 tracker #2.

**Comment by luomiao** [comment]
@govint 
Thank you for the information!
Updated to 17.06.0-ce and problem seems resolved.
